hydpy/models/whmod/whmod_script.py

```diff
@@ -1255,20 +1359,13 @@ def _initialize_whmod_models(
         )
         con.bfi(_return_con_hru(hrus, "bfi")[0])
 
-        verzoegerung = str(_return_con_hru(hrus, "verzoegerung")[0])
+        verzoegerung = _return_con_hru(hrus, "verzoegerung")[0]
```

Comment:
Here (and in some other places), we analyse only the first hydrological response unit of a raster cell.  Is this approach safe?

Reply:
Probably not, but not part of my pull request.